code/_onclick/item_attack.dm

```diff
@@ -118,7 +118,9 @@
 /mob/living/attacked_by(obj/item/I, mob/living/user)
 	send_item_attack_message(I, user)
 	if(I.force)
-		apply_damage(I.force, I.damtype)
+		var/armour_block = run_armor_check(null, MELEE, armour_penetration = I.armour_penetration)
+		armour_block = min(armour_block, 90)
```

Comment:
If we're setting a general upper limit on armor, I think it should be 75% damage reduction. There are not many cases where more than this should be obtainable as this is already bordering on unstoppable even when outnumbered if paired with any kind of healing capability.

Reply:
Could do, this was mirrored from the current implementation which is 90%. I wonder if a smoothstep function that makes it slowly less effective to a max of 75 would be a good idea. Would make getting more armour always better, but still have the limit

Reply:
They probably shouldn't have 70 armour to begin with

Reply:
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/26465327/01e35d54-d055-4942-9fdc-b9da08e1c67c)
This is one possible solution